.github/workflows/ci.yml

```diff
@@ -274,6 +274,86 @@ jobs:
           path: reports-${{ matrix.server }}-base-tests-${{ matrix.tests }}.zip
           if-no-files-found: ignore
 
+  test-fips:
+    name: Base testsuite (fips)
+    needs: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        server: ['bcfips-nonapproved-pkcs12']
+        tests: ['group1']
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+
+      - name: Check whether HEAD^ contains crypto related changes
+        run: echo "GIT_CRYPTO_RELEVANT_DIFF=$( git diff --name-only HEAD^ | egrep -ic -e 'crypto' )" >> $GITHUB_ENV
+
+      - name: Cache Maven packages
+        if: ${{ github.event_name != 'pull_request' || env.GIT_CRYPTO_RELEVANT_DIFF != 0 }}
```

Comment:
I think I know the reasoning behind the condition. However, given how FIPS changes the runtime and how much the server rely on crypto operations, wouldn't be safer to always run the tests?

Reply:
Yes, I was a bit undecided whether to add condition or not and I've added it first. But I've removed it now based on your feedback. We can see if GH action builds are OK and eventually add the condition later if it turns that PR builds are too long...